src/components/app/details/cdDetails/CDDetails.tsx

```diff
@@ -709,19 +708,6 @@ const SelectEnvironment: React.FC<{ environments: AppEnvironment[] }> = ({ envir
         }
     }
 
```

Comment:
Hope this deletion does not make any other impact. Please do confirm

Reply:
no impact